src/zkevm_specs/evm/execution/error_invalid_opcode.py

```diff
@@ -0,0 +1,38 @@
+from ...util import FQ
+from ..execution_state import ExecutionState
+from ..instruction import Instruction, Transition
+from ..opcode import invalid_opcodes
+from ..table import CallContextFieldTag
+
+
+def invalid_opcode(instruction: Instruction):
+    # Fixed lookup for invalid opcode.
+    opcode = instruction.opcode_lookup(True)
```

Comment:
in this case, opcode should not be true code,      `opcode = instruction.opcode_lookup(False)`  should be correct.

Reply:
Fixed to `instruction.opcode_lookup(False)`.